l10n/en/firefox/privacy-hub.ftl

```diff
@@ -54,90 +41,13 @@ firefox-privacy-hub-learn-more-about-our-mission = Learn more about our mission
-
-firefox-privacy-hub-if-you-want-to-see-what = If you want to see what { -brand-name-firefox } is blocking for you, visit this page on your { -brand-name-firefox } desktop browser.
-firefox-privacy-hub-see-what-firefox-has-blocked = See what { -brand-name-firefox } has blocked for you
-
-# "Enhanced Tracking Protection" is a feature name; it should be capitalized
-firefox-privacy-hub-get-enhanced-tracking-protection = Get Enhanced Tracking Protection
-
-firefox-privacy-hub-download-the-firefox-browser = Download the { -brand-name-firefox } browser
-firefox-privacy-hub-update-your-firefox-browser = Update your { -brand-name-firefox } browser
-firefox-privacy-hub-invisible-to-the-top-trackers = Invisible to the top trackers
-firefox-privacy-hub-meet-four-of-the-most-common = Meet four of the most common categories of trackers — who won’t meet you.
-firefox-privacy-hub-always-in-your-control = Always in your control
-firefox-privacy-hub-want-to-customize-what = Want to customize what gets blocked? Your settings are only one click away.
-firefox-privacy-hub-protection-beyond-tracking = Protection beyond tracking
-firefox-privacy-hub-if-you-have-a-mozilla-account = If you have a { -brand-name-mozilla-account }, you can also see how we’re helping you protect your personal info and passwords.
-
-# Obsolete string
-firefox-privacy-hub-if-you-have-a-firefox-account = If you have a { -brand-name-firefox-account }, you can also see how we’re helping you protect your personal info and passwords.
-
-# Variables:
-#   $trackers (number) - localized number total of trackers blocked by Firefox users worldwide
-firefox-privacy-hub-more-than-s-trackers-blocked = More than { $trackers } trackers blocked each day for { -brand-name-firefox } users worldwide
-
-firefox-privacy-hub-mozilla-monitor = { -brand-name-mozilla-monitor }
-
-firefox-privacy-hub-when-you-enter-your-email-v2 = When you enter your email address in { -brand-name-mozilla-monitor }, we forget it immediately after we’ve checked for a match in known data breaches — unless you authorize us to continue monitoring new breaches for your personal information.
-
-firefox-privacy-hub-check-for-breaches = Check for breaches
 firefox-privacy-hub-firefox-lockwise = { -brand-name-firefox-lockwise }
 firefox-privacy-hub-the-passwords-and-credentials = The passwords and credentials you save in { -brand-name-firefox-lockwise } are encrypted on all your devices, so not even we can see them.
 firefox-privacy-hub-firefox-send = { -brand-name-firefox-send }
 firefox-privacy-hub-we-cant-see-the-names-or = We can’t see the names or content of the large files you share through { -brand-name-firefox-send } because they’re encrypted end-to-end — you choose who sees what you send, and you can even set an expiration date and password.
 firefox-privacy-hub-send-a-file = Send a file
```

Comment:
Here all the ftl references to EOLed products (Lockwise, Send etc.) can be removed too?

```suggestion
```